Good Coding Practices/src/js/scripts.js

```diff
@@ -1,41 +1,106 @@
 // piece object
-const piece = (function() {
-  let el = null;
-  const init = function(el) {
+const piece = (function () {
+  // let el = null;
+  const init = function (el) {
     this.el = el;
+    saveDefaultPositin.call(this);
+    setColorByTemperature.call(this);
   };
-  const moveDelta = function(dx, dy) {
+
+  const reset = function () {
+    const posLeft = localStorage.getItem("defaultPosLeft");
+    const posTop = localStorage.getItem("defaultPosTop");

```

Comment:
`localStorage` is overkill for this.
Use it to persist data between page loads, not within a single run.